src/sc_crawler/vendors/alicloud.py

```
@@ -0,0 +1,861 @@
+def _bss_client(
+    region_id: str = environ.get("ALIBABA_CLOUD_REGION_ID", "eu-central-1"),
+) -> BssClient:
+    """Create an Alibaba Cloud BSS client using the default credentials chain."""
+    cred = CredClient()
+    config = Config(credential=cred, region_id=region_id)
+    return BssClient(config)
+
+
+def _get_sku_prices(
+    sku_type: str,
+    extra_request_params: dict = {},
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)